pages/c/_cluster/apps/install.vue

```diff
@@ -921,354 +762,300 @@ export default {
-          <t k="catalog.install.action.ignoreWarning" />
-        </button>
-        <button type="button" class="btn role-primary" @click="cancel">
-          <t k="generic.cancel" />
-        </button>
-      </div>
-    </template>
-
-    <template v-else>
-      <div class="row mb-20">
-        <div class="col span-6">
-          <LabeledSelect
-            :label="t('catalog.install.chart')"
-            :value="chart"
-            :options="charts"
-            :selectable="option => !option.disabled"
-            :get-option-label="opt => getOptionLabel(opt)"
-            option-key="key"
-            @input="selectChart($event)"
+          <NameNsDescription
+            v-if="chart"
+            v-model="value"
+            :mode="mode"
+            :name-disabled="nameDisabled"
+            :name-required="false"
+            :name-ns-hidden="!showNameEditor"
+            :force-namespace="forceNamespace"
+            :namespace-new-allowed="!existing && !forceNamespace"
+            :extra-columns="showProject ? ['project'] : []"
           >
-            <template v-slot:option="opt">
-              <template v-if="opt.kind === 'divider'">
-                <hr />
-              </template>
-              <template v-else-if="opt.kind === 'label'">
-                <b style="position: relative; left: -2.5px;">{{ opt.label }}</b>
-              </template>
+            <template v-if="showProject" #project>
+              <LabeledSelect
+                v-model="project"
+                :disabled="!namespaceIsNew"
+                :label="t('catalog.install.project')"
+                option-key="id"
+                :options="projectOpts"
+                :tooltip="!namespaceIsNew ? t('catalog.install.namespaceIsInProject', {namespace: value.metadata.namespace}, true) : ''"
+                :hover-tooltip="!namespaceIsNew"
+                :status="'info'"
+              />
             </template>
-          </LabeledSelect>
+          </NameNsDescription>
         </div>
-        <div v-if="chart" class="col span-6">
-          <LabeledSelect
-            :label="t('catalog.install.version')"
-            :value="$route.query.version"
-            :options="filteredVersions"
-            :selectable="version => !version.disabled"
-            @input="selectVersion"
-          />
+      </template>
+      <template #helmValues>
+        <p v-if="step2Description" class="row mb-10">
+          {{ step2Description }}
+        </p>
+        <div class="step__values__controls">
+          <!-- Edit as YAML / Back to Form -->
+          <ButtonGroup
+            v-if="(valuesComponent || hasQuestions)"
+            v-model="preShowPreview"
+            :options="formYamlOptions"
+            inactive-class="bg-disabled btn-sm"
+            active-class="bg-primary btn-sm"
+            :disabled="preShowPreview != showPreview"
+          ></ButtonGroup>
+          <!-- Continue Editing / Show Diff -->
+          <ButtonGroup
```

Comment:
Show diff is enabled even when there's no difference now (the button was disabled before so you couldn't get to this state, which leaves you with an odd empty box.

There's an awful lot of btngroup going on here.  6 buttons in 3 groups, each coming & going depending on the last, for what is really a choice of 3-4 views.  Seems like it could be just one btngroup: `[ Edit Options | Edit Yaml | Split Diff | Unified Diff]` or two `[Edit Options | Edit Yaml | Compare Changes] [Split|Unified]`.

![image](https://user-images.githubusercontent.com/753917/117427560-04b4e680-aeda-11eb-8f2a-f931eb0ac484.png)

Reply:
Went with option 2